ui/apps/platform/src/Containers/Dashboard/NamespaceSelect.tsx

```diff
@@ -1,56 +1,29 @@
-import React from 'react';
-import type { ChangeEvent } from 'react';
-import { Divider } from '@patternfly/react-core';
-import { Select, SelectGroup, SelectOption } from '@patternfly/react-core/deprecated';
-import type { SelectOptionObject } from '@patternfly/react-core/deprecated';
+import React, { useState } from 'react';
+import type { MouseEvent, Ref } from 'react';
+import {
+    Badge,
+    Button,
+    Divider,
+    Flex,
+    FlexItem,
+    MenuToggle,
+    Select,
+    SelectGroup,
+    SelectList,
+    SelectOption,
+    TextInputGroup,
+    TextInputGroupMain,
+    TextInputGroupUtilities,
+} from '@patternfly/react-core';
+import type { MenuToggleElement } from '@patternfly/react-core';
+import { TimesIcon } from '@patternfly/react-icons';
 
-import useSelectToggle from 'hooks/patternfly/useSelectToggle';
 import { flattenFilterValue } from 'utils/searchUtils';
 import type { Cluster } from './types';
 
 import './NamespaceSelect.css';
 
-const selectAll: unique symbol = Symbol('Select-All-Namespaces');
-
-function createOptions(clusters: Cluster[], filterValue?: string) {
-    let clustersToShow: Pick<Cluster, 'name' | 'namespaces'>[] = [];
-
-    if (filterValue) {
-        clusters.forEach(({ name: clusterName, namespaces }) => {
-            // If the search filter matches the name of the cluster, include all that cluster's
-            // namespaces.
```

Comment:
This functionality looks to be removed in the refactor - is this intentional?

Reply:
Previously, we allowed you to type in a cluster and then filter to show you the namespaces included in that cluster. But doesn't that seem odd? If you wanted to do that, you could just select that specific cluster in the first dropdown and then see that reflected in the second dropdown. I was a little confused by that UX, so I corrected it here. 

Let me know what you think? cc: @mansursyed @zhenpesky